runtime/include/qio/qio_error.h

```diff
@@ -146,17 +148,18 @@ qioerr qio_mkerror_errno(void);
 #define EFORMAT (EXTEND_ERROR_OFFSET+2)
 #endif
 
-// Make sure we have an EILSEQ
+// These are errors not available on every platform.
 #ifndef EILSEQ
 #define EILSEQ (EXTEND_ERROR_OFFSET+3)
 #endif
-
-// Make sure we have EOVERFLOW
 #ifndef EOVERFLOW
 #define EOVERFLOW (EXTEND_ERROR_OFFSET+4)
 #endif
+#ifndef ENODATA
+#define ENODATA (EXTEND_ERROR_OFFSET+5)
+#endif
 
-#define EXTEND_ERROR_NUM 5
+#define EXTEND_ERROR_NUM 6
```

Comment:
This seems unfortunate and like something a programmer would miss needing to update.  Is there a way to generate it, or update it within each of the ifndefs, so that it's at least obvious that these are connected?